widget/menu_item.go

```diff
@@ -12,7 +12,10 @@ import (
 
 // newMenuItemSeparator creates a separator meant to separate MenuItems.
 func newMenuItemSeparator() fyne.CanvasObject {
-	return widget.NewDivider(fyne.NewSize(1, 2), theme.DisabledTextColor)
+	d := widget.NewDivider()
+	d.MinSz = fyne.NewSize(1, 2)
+	d.ColorFn = theme.DisabledTextColor
```

Comment:
Now that menus are on the same background as everything else, should we not be using a consistent divider color?